packages/fxa-content-server/app/scripts/views/base.js

```diff
@@ -173,6 +173,7 @@ var BaseView = Backbone.View.extend({
     this.user = options.user;
     this.lang = options.lang;
     this.window = options.window || window;
+    this.surveys = options.config ? options.config.surveys : [];
```

Comment:
I don't think we need to attach the surveys to all the views. IIUC the only place that need to examine all the survey is the module that decides if the user get a survey.